reloc.ml

```diff
@@ -89,6 +89,7 @@ let get_output ?(use_bash = false) ?(accept_error=false) cmd =
     else
       begin
         let (cfn, oc) = open_temp_file "longcmd" ".sh" in
+          output_string oc "exec ";
```

Comment:
Is this definitely correct? The command before was going to go via `Sys.command` (i.e. "`sh -c`"-like), which this in theory breaks?

Reply:
The command goes into a bash file that's executed as `bash longcmdxxxxxx.sh` below. As it's always a single invocation (no pipes, no command chains) and sometimes redirections, I think it's exactly the use-case for `exec`. Am I mistaken?